charts/flipt/templates/migration_job.yaml

```
@@ -0,0 +1,69 @@
+{{- if .Values.migration.enabled }}
```

This is the bit that is breaking. It is causing a nil pointer.
Once I make this change locally everything work 👍 
```suggestion
{{- if (.Values.migration).enabled }}
```